src/component/utils/exploration/__tests__/wrapBlockNodes.js

```diff
@@ -0,0 +1,445 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+ui_infra
+ * @format
+ */
+
+'use strict';
+
+jest.disableAutomock();
+
+const React = require('react');
+const UnicodeBidiDirection = require('UnicodeBidiDirection');
+const CompositeDraftDecorator = require('CompositeDraftDecorator');
+const ContentState = require('ContentState');
+const EditorState = require('EditorState');
+const BlockTree = require('BlockTree');
+const ContentBlockNode = require('ContentBlockNode');
+const ReactTestRenderer = require('react-test-renderer');
+const wrapBlockNodes = require('wrapBlockNodes');
+const TestHelper = require('_DraftTestHelper');
+const DraftEditorBlockNode = require('DraftEditorBlockNode.react');
+const {List} = require('immutable');
+const blockRenderMap = require('DefaultDraftBlockRenderMap');
+const DraftOffsetKey = require('DraftOffsetKey');
+const DefaultDraftBlockRenderMap = require('DefaultDraftBlockRenderMap');
+
+const assertWrapBlockNodes = (contentState, extraProps = {}) => {
+  const nodes = contentState.getBlocksAsArray().map(block => {
+    const configForType =
+      blockRenderMap.get(block.getType()) || blockRenderMap.get('unstyled');
+
+    const wrapperTemplate = configForType.wrapper;
+    const key = block.getKey();
+    const editorState = EditorState.createWithContent(contentState);
+    const offsetKey = DraftOffsetKey.encode(key, 0, 0);
+
+    const childProps = {
+      blockRenderMap: DefaultDraftBlockRenderMap,
```

Comment:
We could move this outside of this function and maybe called it something like `DEFAULT_CHILD_PROPS` and inside here we could simply do a:

`
const childProps = {
   ...DEFAULT_CHILD_PROPS,
   ...props
}
`